kafka-versions

```diff
@@ -25,4 +25,5 @@
 2.0.0              2.0     2.0   b28e81705e30528f1abb6766e22dfe9dae50b1e1e93330c880928ff7a08e6b38ee71cbfc96ec14369b2dfd24293938702cab422173c8e01955a9d1746ae43f98
```

Comment:
We have https://github.com/strimzi/strimzi-kafka-operator/issues/1426 to discuss, but I think we need to do something soon about this ever growing list.

Reply:
Well, I asked about that above ... so ... what is your opinion?